src/components/about-us/section-01/index.js

```diff
@@ -0,0 +1,147 @@
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import { screen } from '../utils/screen'
+import { content } from '../constants/data/section1Content'
+import Title from './title'
+
+const containerWidth = {
+  mobile: '100%',
+  tablet: '719px',
+  desktop: '1024px',
+  overDesktop: '1440px'
+}
+
+const ContainerWrapper = styled.div`
+  position: relative;
+  margin: 0 auto;
+  background-color: white;
+  overflow: hidden;  
+  ${screen.desktopAbove`
+    width: ${containerWidth.desktop};
+  `}
+  ${screen.overDesktop`
+    width: ${containerWidth.overDesktop};
+  `}
+  ${screen.tablet`
+    width: ${containerWidth.tablet}
+  `}  
+`
+
+const SectionBlock = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  ${screen.mobile`
+    flex-direction: column;
+    align-items: center;
+  `}
+`
+
+const Oneblock = styled.div`
+  text-align: center;
+  width: 30%;
+  padding: 10px;
+  min-height: 50vh;
+  img{
+    width: 100%;
+    padding: 20px;
+  }
+  h2{
+    font-size: 28px;
+    margin-bottom: 0;
+  }
+  h3{
+    font-size: 20px;
+    margin-top: 0;
+  }
+  p.description{
+    font-weight: bold;
+    font-size: 16px;
+  }
+  p.engDescription{
+    font-weight: bold;
+    font-size: 14px;
+  }
+  ${screen.tablet`
+    p{
+      font-size: 12px;
+    }
+  `}
+
+  ${screen.mobile`
+    width: 90%;
+    height: 100vh;
+    border-bottom: solid 0.5px #a67a44;
+    display: flex;
```

Comment:
why need `flex` on mobile version?

Reply:
I used `flex` on mobile version to keep the content horizontal and vertical centered inside a feature block `FeatureBlock`.